bug_reports.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Creating Useful Bug Reports
+permalink: /bug_reports/
+---
+
+The instructions in this document detail the current steps required to provide the most actionable bug reports for crashes and other bad behavior. Each step is optional but will greatly improve how quickly issues are diagnosed and addressed.
+
+## Create an Issue on Github
+* A new Github issue may be created at https://github.com/flutter/flutter/issues/new
+
+## Provide the Flutter Version
+* Run `flutter --version` in your project and paste the results into the Github Issue
+![flutter version](/images/flutter_version.png)
+
+## Run the Command in Verbose Mode
+* All Flutter commands accept the `--verbose` flag. If attached to the issue, the output from this command may aid in diagnosing the issue.
```

Comment:
Add "If your problem is with the 'flutter' tool, then:" or some such (since people attaching verbose "flutter run" output to every framework bug is going to get old fast)